code/modules/ventcrawl/ventcrawl.dm

```diff
@@ -35,8 +35,8 @@ var/list/ventcrawl_machinery = list(
 
 /mob/living/proc/is_allowed_vent_crawl_item(var/obj/item/carried_item)
 	for(var/type in can_enter_vent_with)
-		if(istype(carried_item, can_enter_vent_with))
-			return get_inventory_slot(carried_item) == 0
+		if(is_type_in_list(carried_item, can_enter_vent_with))
```

Comment:
just a note that you're looping over `can_enter_vent_with` uselessly here; you either want `istype(carried_item, type)` or to remove the loop as Psi did in Baystation12/Baystation12#14322